tests/puppeteer/campaigns/smoke/crawlingBO.js

```diff
@@ -0,0 +1,65 @@
+// importing pages
+const LOGIN_PAGE = require('../../pages/BO/login');
+const DASHBOARD_PAGE = require('../../pages/BO/dashboard');
+const COMMON_PAGE = require('../../pages/BO/commonPage');
+const ORDERS_PAGE = require('../../pages/BO/orders');
+const PRODUCTS_PAGE = require('../../pages/BO/products');
+const CUSTOMERS_PAGE = require('../../pages/BO/customers');
+
+let page;
+let LOGIN;
+let DASHBOARD;
+let COMMON;
+let ORDERS;
+let PRODUCTS;
+let CUSTOMERS;
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  LOGIN = await (new LOGIN_PAGE(page));
```

Comment:
```suggestion
  loginPage = await (new LoginPage(page));
```
Using the new conventions.